netctl/netctl.go

```diff
@@ -401,3 +402,14 @@ func dumpList(ctx *cli.Context, list []map[string]interface{}) {
 	os.Stdout.Write(content)
 	os.Stdout.WriteString("\n")
 }
+
+func showVersion(ctx *cli.Context) {
```

Comment:
but we have not added API version yet (or to be specific objmodel version), right? 

Also showing the version here doesn't ensures compatibility. Those compat checks would need to happen in the client/server protocol itself and need to be designed.

Anyways I am not arguing we don't need versioning or something similar. I am just saying whatever we are adding right now (on top of a basic version flag) is not sufficient/complete as is since we have not figured the things like compat checks etc yet.